helpers/changesLogGenerator.js

```diff
@@ -0,0 +1,34 @@
+const { getTitle } = require('../constants');
+
+let issuesNumbersIncluded = [];
+
+function checkForDuplication(issue, duplicateIssues) {
+  if(duplicateIssues === true) return `\n- ${issue.title} [${issue.number}]`;
+
+  const duplicate = issuesNumbersIncluded.some(numberIncluded => (numberIncluded === issue.number));
```

Comment:
As prevous, IMO good convention is to prefix boolean variables with `is`:

```suggestion
  const isDuplicate= issuesNumbersIncluded.some(numberIncluded => numberIncluded === issue.number);
```

Reply:
I will add this as part of a single commit